src/metabase/lib/schema/metadata.cljc

```diff
@@ -256,6 +268,10 @@
    [:description {:optional true} [:maybe ::lib.schema.common/non-blank-string]]])
 
 (mr/def ::legacy-metric
+  "Malli schema for a legacy v1 [[metabase.models.legacy-metric]], but with kebab-case keys. A Metric defines an MBQL
+  snippet with an aggregation and optionally a filter clause. You can add a `:metric` reference to the `:aggregations`
```

Comment:
Should that be `:legacy-metric`?

Reply:
It's talking about `:metric` clauses which are still called `:metric` AFAIK even if they are V1 metrics